### ISSUE TITLE
❌ Hello Theme, Elementor 3.8.0 - Loop item - Change Custom Post tittle and Post excerpt issue  

### ISSUE THREAD

**Issue by diehendrick** [elementor/elementor#20284]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

I am using Hello theme from elementor, and elementor pro,   it has two issue I found. from my side.    when I created loop item..
Elementor pro 3.8.0

1. Post title if I add POST URL, the font is not able to customize. 
2. Post exerpt is not working, even I already set exerpt length. 

### Post title 
1.  Theme build
2.  Create template component loop item. 
3. add post title. 
4.  in input link, add POST URL. 
5. in style tab: set custom typography. 
![image](https://user-images.githubusercontent.com/1661591/200128790-ffa12baa-2ceb-4191-aaac-0057c7408266.png)

### Post excerpt 
1.  Theme build
2.  Create template component loop item. 
3. add Post excerpt 
4.  set excerpt length to 250. 
5. in preview mode, cannot see the post content. 
![image](https://user-images.githubusercontent.com/1661591/200128801-705018f8-ca6e-4b52-a53e-50fbae915f5b.png)

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [ ] I can reproduce this bug consistently using the steps above.

### System Info

<html>
<body>
<div class="elementor-system-info-section elementor-system-info-server" style="outline: 0px; margin-bottom: 10px; color: rgb(60, 67, 74); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Roboto, Oxygen-Sans, Ubuntu, Cantarell, &quot;Helvetica Neue&quot;, sans-serif; font-size: 13px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(240, 240, 241); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">

Server Environment |   |  
-- | -- | --
Operating System: | Linux |  
Software: | LiteSpeed |  
MySQL version: | MariaDB Server v10.5.17 |  
PHP Version: | 7.4.30 |  
PHP Memory Limit: | 512M |  
PHP Max Input Vars: | 5000 |  
PHP Max Post Size: | 128M |  
GD Installed: | Yes |  
ZIP Installed: | Yes |  
Write Permissions: | All right |  
Elementor Library: | Connected |  

</div>
</body>
</html>

<html>
<body>

WordPress Environment |   |  
-- | -- | --
Version: | 6.1 |  
Site URL: | https://jadidesigner.com |  
Home URL: | https://jadidesigner.com |  
WP Multisite: | No |  
Max Upload Size: | 128 MB |  
Memory limit: | 40M |  
Max Memory limit: | 512M |  
Permalink Structure: | /%postname%/ |  
Language: | en-US |  
Timezone: | 0 |  
Admin Email: | dei@theprojekts.com |  
Debug Mode: | Inactive

</body>
</html>

<html>
<body>

Theme |   |  
-- | -- | --
Name: | Hello Elementor |  
Version: | 2.6.1 |  
Author: | Elementor Team |  
Child Theme: | No

</body>
</html>

**Comment by nicholaszein** [comment]
Hey @diehendrick, 

Thank you for submitting your issue! 🙏 

### ❌ While reviewing your report, we noticed it was missing some key pieces of information.

- **Prerequisites:** please search the repository before submitting an issue. 
  - The question about the Post Excerpt was already asked here: #20256.
- **Isolating the Problem:** You need to make sure this only happens while using Elementor, Elementor Pro, and the Hello Elementor theme. We do not provide support for 3rd-party themes and plugins.
- **System info:** You did not provide any information about your [System Info](https://elementor.com/help/systems-info/) as outlined in our [guidelines](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml). We need Elementor's System Info.
- **One issue per thread:** We can't track multiple issues per thread. Open a new issue for each request.

Without them, we simply can't help. 😕 

👉 Please follow the guidelines for opening a new [🐞 Bug Report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml), so we can help you better.


### ⚠️ Note that GitHub issues are reserved for Elementor 🐞 Bug Reports only, and should not be used for support.

❔ If you have questions or need general support, please use one of the following channels:

- [Elementor Global Community](https://facebook.com/groups/Elementors/) on Facebook
- [Elementor Help Center](https://docs.elementor.com)
- [Support Forum](https://wordpress.org/support/plugin/elementor) on WordPress.org 
- [Elementor Developers Documentation](https://developers.elementor.com/)

💎 If you have purchased [Elementor Pro](https://elementor.com/pro), you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.


#### For  💡 **New Ideas** or 🚀 **Feature Requests** please [create a new Discussion](https://github.com/elementor/elementor/discussions/new) in the [Discussions](https://github.com/elementor/elementor/discussions) tab.

Best regards

**Comment by nicholaszein** [comment]
Hello @diehendrick!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro **v3.13.0**! 🥳 

You can now use the Post Excerpt widget and Dynamic Tag, and switch on the option to `Apply to post content` to include part of your post content in case you don't have a post excerpt.

![image](https://user-images.githubusercontent.com/113993460/236926863-e290e71c-f2e3-43c3-ae48-aa24bdcc94b7.png)
----

✅ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
